engine/proto/errorv2.proto

```diff
@@ -4,9 +4,43 @@ package enginepb;
 
 option go_package = "github.com/pingcap/tiflow/engine/enginepb";
 
-message ErrorV2 {
-  string name = 1;
-  bytes details = 2;
-  string stack_trace = 3;
-}
+// NOTE: for all openapi error output, please use grpc.Status as the response body json format
+// HTTP error ouput example:
+//
+// HTTP/1.1 500 OK
+// Content-Length: 88
+// Content-Type: application/json
+// Connection: Closed
+// 
+//{
+//   "Code": ${INTERNAL},
+//   "Message":"job inner error",
+//   "Details":[
+//      {
+//         "Code": ${BusinessErrorCode},
+//         "Name": "DFLOW:ErrJobManagerReadRespBodyFail"
+//         "Message":"business error",
+//         "Details":[
+//            "business stack",
+//         ]
+//      }
+//   ]
+//}
 
+// Status is defined for inner status(engine and business)
+// It is used as a part of enginepb.Job and grpc.Status.Details
+// refer to: https://cloud.google.com/apis/design/errors
+message Status {
```

Comment:
What's the relationship between this `Status` and what's documented in https://cloud.google.com/apis/design/errors?

Reply:
This pr will split out the `response format` related code.